modules/installation-nutanix-config-yaml.adoc

```diff
@@ -0,0 +1,93 @@
+
+You can customize the `install-config.yaml` file to specify more details about your {product-title} cluster's platform or modify the values of the required parameters.
+
+[IMPORTANT]
+====
+This sample YAML file is provided for reference only. You must obtain your `install-config.yaml` file by using the installation program and modify it.
+====
+
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: example.com <1>
+compute: <2> <3>
+- hyperthreading: Enabled <4>
+  name: worker
+  platform: {}
+  replicas: 3
+controlPlane: <2> <3>
+  hyperthreading: Enabled <4>
+  name: master
+  platform: {}
+  replicas: 3
+metadata:
+  creationTimestamp: null
+  name: test-cluster <1>
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: 10.0.0.0/16
+ifndef::openshift-origin[]
+  networkType: OpenShiftSDN
+endif::openshift-origin[]
+ifdef::openshift-origin[]
+  networkType: OVNKubernetes
+endif::openshift-origin[]
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  nutanix:
+    apiVIP: 10.40.142.7 <1>
+    ingressVIP: 10.40.142.8 <1>
+    password: samplepassword <1>
+    port: "9440" <1>
+    prismCentral: your.prismcentral.domainname <1>
+    prismElementUUID: 0005b0f1-8f43-a0f2-02b7-3cecef193712
+    subnetUUID: c7938dc6-7659-453e-a688-e26020c68e43
+    username: sampleadmin
+credentialsMode: Manual
+publish: External
+pullSecret: '{"auths": ...}' <1>
+ifndef::openshift-origin[]
+fips: false <5>
+sshKey: ssh-ed25519 AAAA... <6>
+endif::openshift-origin[]
+ifdef::openshift-origin[]
+sshKey: ssh-ed25519 AAAA... <5>
+endif::openshift-origin[]
+----
+<1> Required. The installation program prompts you for this value.
+<2> If you do not provide these parameters and values, the installation program provides the default value.
+<3> The `controlPlane` section is a single mapping, but the compute section is a sequence of mappings. To meet the requirements of the different data structures, the first line of the `compute` section must begin with a hyphen, `-`, and the first line of the `controlPlane` section must not. Although both sections currently define a single machine pool, it is possible that future versions of {product-title} will support defining multiple compute pools during installation. Only one control plane pool is used.
+<4> Whether to enable or disable simultaneous multithreading, or `hyperthreading`. By default, simultaneous multithreading is enabled to increase the performance of your machines' cores. You can disable it by setting the parameter value to `Disabled`. If you disable simultaneous multithreading in some cluster machines, you must disable it in all cluster machines.
++
+[IMPORTANT]
+====
+If you disable simultaneous multithreading, ensure that your capacity planning accounts for the dramatically decreased machine performance. Use larger machine types, such as `n1-standard-8`, for your machines if you disable simultaneous multithreading.
```

Comment:
I do not believe there are machine types defined for nutanix.

Reply:
@rvanderp3 While machine types might not be defined for Nutanix, is there anything else we might suggest to help mitigate degraded performance if multithreading is disabled?

Reply:
Done.